Formula/pdnsrec.rb

```diff
@@ -12,40 +12,37 @@ class Pdnsrec < Formula
     sha256 "a9faf9edf0e71de5e9ec9fa70d27811f353a8451b41746208fbdc0d592aa5910" => :mountain_lion
   end
 
-  depends_on :macos => :lion
+  depends_on "pkg-config" => :build
   depends_on "boost"
-  depends_on "lua" => :optional
+  depends_on "openssl"
+  depends_on "lua"
+  depends_on "gcc" => :linked if MacOS.version <= :mavericks
```

Comment:
> Let's kill this entire line.

It actually links to `gcc` in the bottle, so I'm not sure ditching the dep was wise. It may cause some of the `linkage` issues we'd discussed before:

```
@@HOMEBREW_PREFIX@@/opt/gcc/lib/gcc/6/libstdc++.6.dylib
```